e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-continue-no-errors.spec.js

```diff
@@ -59,4 +68,16 @@ describe("Insurance - Your business - Company details page - As an Exporter I wa
       cy.url().should('eq', natureOfBusinessUrl);
     });
   });
+
+  describe('when resubmitting company number on company details page', () => {
+    before(() => {
+      partials.backLink().click();
+      submitButton().click();
+      partials.backLink().click();
```

Comment:
Could this be simpler? Seems it's in a bit of a loop here - back, submit, back?

Reply:
I can navigate to url if that's easier, will be doing the same thing though

Reply:
Could it not just be one line of `partials.backLink().click();` ?

Reply:
it is on the nature of the business page so I need to get back to company details, and then it redirects again so need to go back again